rfcs/999-introduce-release-controller.adoc

```diff
@@ -0,0 +1,126 @@
+= 999 Introduce Release Controller
+Igor Sutton <igor.sutton@booking.com>
+1970-01-01
```

Comment:
I think we should either kill this metadata or have it be "real". Bit funky to have a bunch of '999-foo' and "standards track" things.